modules/custom/activity_send/modules/activity_send_email/src/Plugin/EmailFrequency/Immediately.php

```diff
@@ -25,15 +24,10 @@ class Immediately extends EmailFrequencyBase {
   /**
    * {@inheritdoc}
    */
-  public function processItem(Activity $activity, Message $message, User $target) {
-    if ($target->isBlocked()) {
```

Comment:
This needs to stay, sure to be sure. Also to ensure we have backwards compatibility and imagine in the meantime someone got blocked we also want to ensure we filter that out at the end.

Reply:
Okay.